ui/pages/home/home.component.js

```diff
@@ -46,6 +46,17 @@ const LEGACY_WEB3_URL =
 const INFURA_BLOCKAGE_URL =
   'https://metamask.zendesk.com/hc/en-us/articles/360059386712';
 
+function shouldCloseNotificationPopup({
+  isNotification,
+  totalUnapprovedCount,
+  isSigningQRHardwareTransaction,
+}) {
+  return (
+    isNotification &&
+    totalUnapprovedCount === 0 &&
+    !isSigningQRHardwareTransaction
+  );
+}
```

Comment:
nit: breathing space
```suggestion
}

```

Reply:
Done!